apps/minifront/src/components/layout.tsx

```diff
@@ -5,19 +5,24 @@ import { Toaster } from '@repo/ui/components/ui/toaster';
 import { Footer } from './footer/footer';
 import '@repo/ui/styles/globals.css';
 import { MotionConfig } from 'framer-motion';
+import { PenumbraProvider } from '@penumbra-zone/react';
+
+const praxOrigin = 'chrome-extension://lkpmkhpnhknhmibgnmmhdhgdilepfghe';
 
 export const Layout = () => {
   return (
-    <MotionConfig transition={{ duration: 0.1 }}>
-      <HeadTag />
-      <div className='flex min-h-screen w-full flex-col'>
-        <Header />
-        <main className='flex size-full flex-1 px-4'>
-          <Outlet />
-        </main>
-        <Footer />
-      </div>
-      <Toaster />
-    </MotionConfig>
+    <PenumbraProvider providerOrigin={praxOrigin}>
```

Comment:
question: does setting `providerOrigin` to Prax mean users can't connect another wallet or that they have to create a state variable that's keeping an dynamic origin for that purpose?

Reply:
minifront should implement UI for the user to select a preferred wallet. currently minifront only supports prax and larger change is out of scope